pages/ActivityStreams/Overview.md

```diff
@@ -33,22 +33,347 @@ These deviations from the standard will be thoroughly documented here.
+Video objects linking to live streaming content may use the string `"Live"` for the duration field or omit it entirely.
+
+Video objects may include a `"preview"` field with an activity sub-object of type link, a URL string or an array of sub-objects and/or strings referring to shorter video files or image files provided as a preview of longer video files.
+It is recommended that implementers at minimum include one image previews wherever possible.
+Preview images should generally include at least one file of the following resolutions: `320x240`, `640x480` or `1024x768`.
+
+Clients that support video objects should also include fields of `"height"` and `"width"` each with positive integer value on either the root activity object or the individual link sub-objects if multiple are present.
+If these fields are not provided, clients may attempt to infer the media type from the headers of the linked file or simply ignore the object.
+Clients may also ignore video files of exceptionally large or small sizes, however videos with a preview of one of the prior mentioned resolutions must always be supported.
+
+For example, the following would be valid video objects:
+
+```json
+{
+  "@context": "https://www.w3.org/ns/activitystreams",
+  "type": "Video",
+  "name": "Puppy Plays With Ball",
+  "url": "http://example.org/video.mkv",
+  "duration": "PT2H"
+}
+```
+
+```json
+{
+  "@context": "https://www.w3.org/ns/activitystreams",
+  "type": "Video",
+  "name": "Big Buck Bunny",
+  "url": {
+    "type": "Link",
+    "href": "https://upload.wikimedia.org/wikipedia/commons/c/c0/Big_Buck_Bunny_4K.webm"
+  },
+  "duration": "PT10M32S",
+  "mediaType": "video/webm",
+  "width": 4000,
+  "height": 2250
+}
+```
+
 ## Extensions
 
 In addition to the standards defined by the Activity Streams specifications, some extensions of Activity Streams are also required by the DSNP due to its decentralized context.
 
-### Required Keys and Values
+### Content Contexts
+
+Clients may sometimes want to represent content as existing within a specific presentation or social context, such as Reddit's subreddit concept, Slack's channels or old-style forum rooms.
+It is recommended that implementers represent this context in activity objects with the `"context"` field.
+If present, this field must have a string value starting with the name of the implementing service followed by forward slash and a string unique to the specific context.
+Implementers are encouraged to add additional forward slash separators to their unique string to represent contexts within contexts.
+For example, a `#development` channel context within a `LibertyDSNP` organization on Slack could be represented by the string, `"Slack/LibertyDSNP/#development"`.
+
+Clients may choose to ignore activity objects with contexts outside the user's current context, but it is strongly recommended that all contexts be made accessible to the user through some form of navigation.
+
```

Comment:
Close to contexts, we should likely go ahead an specify and support [tags](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-tag) and therefore Mentions ala https://www.w3.org/TR/activitystreams-vocabulary/#microsyntaxes

Reply:
We also [support tags](https://github.com/LibertyDSNP/spec/pull/90/files#diff-9213ad9daeed82825a98eec5ea7f3c9e7a48c946b6b372b46e5d1e3f50d24cb7R93) but this is a different use case. Tags are many-to-many, which is great for drawing other user's attention to stuff, but it's not great for chat-room-style conversations like Slack or Discord channels. Contexts are stricter in that a post can only ever have one or none.